Mathlib/Data/Real/CauSeqCompletion.lean

```diff
@@ -24,173 +24,170 @@ open CauSeq
 
 section
 
-parameter {α : Type _}[LinearOrderedField α]
+variable {α : Type _} [LinearOrderedField α]
 
-parameter {β : Type _}[Ring β]{abv : β → α}[IsAbsoluteValue abv]
+variable {β : Type _} [Ring β] {abv : β → α} [IsAbsoluteValue abv]
 
 /-- The Cauchy completion of a ring with absolute value. -/
-def CauchyCat :=
+def Cauchy :=
```

Comment:
I think instead of calling this `CauSeq.Completion.Cauchy` we could just call it `CauSeq.Completion`.  The mathlib3 name was weird anyway.